Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM maven:latest
```

Comment:
might be better to use an alpine version for the lower image size
https://hub.docker.com/_/maven